.editorconfig

```diff
@@ -263,6 +263,9 @@ ij_java_variable_annotation_wrap = normal
 ij_java_wrap_first_method_in_call_chain = true
 # ij_java_wrap_long_lines = false
 
+[*.out]
+insert_final_newline = false
```

Comment:
this is very annoying thing

by default at the very beginning of this file there is 
`insert_final_newline = true`
for all file types

as a result every time it inserts a new line for every file..
Since *.out files are used to compare file content then extra new line symbol might bring difference